tests/reports/test_fights.py

```diff
@@ -74,6 +74,19 @@ def test_events(self) -> None:
         self.assertIsNotNone(events)
         self.assertEqual(len(events), 8)
 
+    def test_event_with_filter_expression(self) -> None:
+        '''
+        The client should be able to handle some user defined filter expressions.
+        '''
+        events = self.fight.events({
+            'filterExpression':{
+                'type':'limitbreakupdate'
```

Comment:
Add an additional filter to test that logical and on the filter is functional

e.g.
```python
events = self.fight.events({
    'filterExpression':{
        'type':'damage',
        'rawDamage':'>=1000',
    }
})
```